AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/logger/appender/MessagerAppender.java

```diff
@@ -66,4 +85,110 @@ private Kind resolveKind(Level level) {
 		return Kind.OTHER;
 	}
 
+	private class Message {
```

Comment:
This class could be `static`.

Reply:
only if we pass the processingEnv as it is used in getElement()

Reply:
Okay, leave it.